packages/manager/apps/telecom/src/app/telecom/telephony/repayments/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,21 @@
+{
+  "telephony_repayments_available_title": "Reversements disponibles",
+  "telephony_repayments_costs_title": "Coûts à verser",
+  "telephony_repayments_costs": "Coûts",
+  "telephony_repayments_grid_buyAmount.value": "Reversement",
+  "telephony_repayments_grid_calledNumber": "Numéro spécial",
+  "telephony_repayments_grid_callingNumber": "Numéro entrant",
+  "telephony_repayments_grid_duration": "Durée (sec.)",
+  "telephony_repayments_grid_startDate": "Date",
+  "telephony_repayments_grid_status_CREATED": "En cours de reversement",
+  "telephony_repayments_grid_status_ERROR": "Erreur sur le reversement",
+  "telephony_repayments_grid_status_NONE": "Aucun reversement",
+  "telephony_repayments_grid_status_PAID": "Reversé",
+  "telephony_repayments_grid_status": "Statut",
+  "telephony_repayments_iban_link": "Modifier mes coordonnées bancaires",
+  "telephony_repayments_message_above_threshold": "Votre prochain reversement sera de <strong>{{ amount }}</strong>",
+  "telephony_repayments_message_bank_account": "Mes coordonnées bancaires",
+  "telephony_repayments_message_under_threshold": "Le montant minimum de <strong>{{ amount }}</strong> n'est pas encore atteint. Aucun reversement n'est prévu à ce jour",
```

Comment:
Same as here:
https://github.com/ovh/manager/pull/6910/files#r866613891

Reply:
It's applied